Assets/VRTK/Scripts/Presence/VRTK_BodyPhysics.cs

```diff
@@ -169,6 +169,8 @@ public enum FallingRestrictors
         protected string footColliderContainerNameCheck;
         protected const string BODY_COLLIDER_CONTAINER_NAME = "BodyColliderContainer";
         protected const string FOOT_COLLIDER_CONTAINER_NAME = "FootColliderContainer";
+        private bool enableBodyCollisionsStartingValue;
```

Comment:
Make these protected. All VRTK stuff should be at least protected now to allow people to extend scripts.